nixos/modules/services/mail/opendkim.nix

```diff
@@ -91,6 +104,22 @@ in {
 
   config = mkIf cfg.enable {
 
+    services.opendkim.settings = {
```

Comment:
Can we set a default UMask of 002 here? That way the socket created by the daemon will be accessible by its group (which we also set by default). See #27260

Reply:
Sounds good, did that